public/desktop/index.html

```diff
@@ -13,13 +13,34 @@
     <link rel="stylesheet" href="../css/draggable.css">
 
   </head>
-  <body>
-
+  <body id="desktop"> 
+    <div class="login">
+      <div class="login-header">
+        <span class="white-txt"> Login </span>
+      </div>
+      <div class="login-form">
+        <form action="POST"></form> 
```

Comment:
u can remove this line, the <form action> is used when submitting data to a php file in the backend, but our REST API is being built w/JavaScript (not php) so this isn't really necessary